internal/proto/codec/sql_error_codec.go

```diff
@@ -54,7 +54,7 @@ func DecodeSqlError(frameIterator *proto.ForwardFrameIterator) isql.Error {
 	}
 	CodecUtil.FastForwardToEndFrame(frameIterator)
 
-	return isql.Error{
+	return sql.Error{
```

Comment:
With 5.0, server also gives suggestion about the error which is in the second frame. https://github.com/hazelcast/hazelcast-python-client/pull/473 

It would be great to add that while implementing the sql feature.

Reply:
Thanks for pointing that out! I think we covered the functionality.